### DISCUSSION TITLE
How many active developers still working on SageTV?

### DISCUSSION THREAD

**Discussion by techiegirl2day** [OpenSageTV/Community#46] [Developer's Q&A]
Hi,
 
    I thought I would ask here how many developers are still working on and or maintaining code on this project? 
 It is starting to feel like the phrase... last one out turn off the lights & close the door applies here. After moving the forum to github it feels like the last nails are going on the coffin lid. 
 Is it down to Joss, Ken & the invisible Jeffrey?

Galaxysurfer

**Comment by jusjoken** [comment]
I am still here.  Due to family related support I am providing I am not currently actively writing code except where fixes are requested/needed for weather, gemstone, phoenix, bmt etc.  I monitor the forums daily but do see a huge decline since the move away from the regular forums.  We are creatures of habit and the move makes communication awkward and perhaps that's one of the reasons for the decline.  Hopefully we find all those users whom have not moved yet but growth will be difficult with this forum move (IMHO).

---
@jusjoken
If you wish to see what I am up to and support my efforts visit my [Patreon page](https://www.patreon.com/user?u=58035567)

**Comment by JustFred51** [comment]
Although I haven't contributed as much development as Ken and others, I continue to make minor bug fixes and don't intend to abandon Sage.  I, too, feel that moving the forum to github hasn't been very positive in terms of new postings.

**Reply by eelpout** [reply]
while I appreciate the effort that went to in setting this place up, it's super awkward to use. :/

**Comment by Sagetvmaster** [comment]
It really does seem that people haven't been able to figure this place out (me included - still learning Github), or they don't know where it went.

**Comment by BobPhoenix** [comment]
Me too.  Just connected to the old forum and saw the message.  Glad I did find the move when I did.

**Reply by enternoescape** [reply]
I haven't been checking in nearly as much as I used to. I too am glad I to discovered where the dialogs went. I have so much I started and I already know inevitably I will never finish. It's harder to find the motivation when I'm barely even using it anymore.

**Comment by Gary0201** [comment]
One thing in BMT that has always annoyed me is when I drop video files into a Sage Video folder, Sage then eventually recognizes it and creates a properties file, then I come along with BMT and flag it as a SageTV recording as a Movie (e.g. I use PlayOn to stream/record something off of Hulu or Netflix).  VMT always inserts 4 characters in front of the description.  To fix that, I clear the flag for being a Sage recording, then set it back.  Those 4 characters are then removed and the entire description is then wrapped in quotes.

**Comment by techiegirl2day** [comment]
Thanx to those that responded to my Post.  I was mainly wanting a headcount of active developers here. Also looking to see what kind of interest there would be to update SageTV to compete with the big boys, if it is even viable.  After that would be what would we need on financial end to get the band back together, and or recruit new blood that can help with the task.  I personally would like to see a new & improved version of SageTV with possibly Apple support etc. The key to all this is doing it in conjunction with the Content providers etc so they don't try to torpedo our efforts, or maybe what is called for is the opposite, meaning stay under the radar.   
     The intent is not to support illegal streaming etc but to come up with an aggregator that includes all that we like to use in one location. No more juggling multiple devices or like the cellphone world does, one for personal & one for business analogy. The push seems to be towards streaming, so maybe we need to focus a bit more on that as in  have something that can be the best of both dvr access & streaming without all the crazy pay to play fees they want to ding us with.  This individual app for each guy thing is becoming worst than the cable subscriptions we were abandoning to regain content control as well as reasonable cost. Maybe it isn't possible?  

Galaxysurfer /techiegirl2day

**Reply by techiegirl2day** [reply]
Would finding a different solution to the replacement forum help our cause? 
 Github is mainly for the Techies. ( Don't really consider myself one, but hopped onboard to troubleshoot & fix my various software programs in use needing code help. )  

Should we look to a different forum that is more accessible & user friendly for everyone?  

Someone please remind me again what rules Google has put into play.  

Should we consider a SageTV fork to revamp the works for modern streaming era?

Galaxysurfer
